charts/hedera-network/config-files/otel-collector-config.yaml

```
@@ -18,9 +18,19 @@ exporters:
   prometheus:
     endpoint: "0.0.0.0:8889"
     const_labels:
-      label1: "{{ .otelDefaults.nameOverride }}-label1"
+      source: "p-{{ .otelDefaults.nameOverride }}" # PromQL: {source='p-otel-collector'}
+  {{- if eq .otelDefaults.prometheusRemoteWrite.enable "true" }}
+  prometheusremotewrite:
+    endpoint: "{{ .otelDefaults.prometheusRemoteWrite.endpoint }}"
+    tls:
+      insecure: true
+    external_labels:
+      source: "prw-{{ .otelDefaults.nameOverride }}" # PromQL: {source="prw-otel-collector"}
```

This should be defaulted to `.Release.Name` if `.otelDefaults.nameOverride` is not set. 

```suggestion
      source: "prw-{{ default .Release.Name .otelDefaults.nameOverride }}" # PromQL: {source="prw-otel-collector"}
```